packages/sidebar-settings/src/blocks/dropdown.ts

```diff
@@ -22,6 +22,7 @@ export type DropdownBlock<AppBridge> = {
 
     /**
      * The size of the dropdown.
+     * @deprecated `size` has no effect anymore, will be removed 13.09.2025 
```

Comment:
If it has no effect already, it can be removed sooner (like in a month)

Reply:
True we can remove it here earlier, just need to make sure its still allowed within the web-app